articles/iot-edge/tutorial-deploy-machine-learning.md

```diff
@@ -36,16 +36,16 @@ The Azure Machine Learning module does not support ARM processors.
 
 Have the following prerequisites on your development machine: 
 * An Azure Machine Learning account. Follow the instructions in [Create Azure Machine Learning accounts and install Azure Machine Learning Workbench](../machine-learning/service/quickstart-installation.md#create-azure-machine-learning-services-accounts). You do not need to install the workbench application for this tutorial. 
-* Module Management for Azure ML on your machine. To set up your environment and create an account, follow the instructions in [Model management setup](../machine-learning/desktop-workbench/deployment-setup-configuration.md).
+* Model Management for Azure ML on your machine. To set up your environment and create an account, follow the instructions in [Model management setup](../machine-learning/desktop-workbench/deployment-setup-configuration.md). You do not need to complete the cluster deployment step.
```

Comment:
People might misinterpret this as saying that they can skip the whole environment setup, which I don't think is the case? Still investigating. We should say something like choose the local deployment instead of the cluster deployment if possible.